bigquery_etl/glean_usage/templates/baseline_clients_first_seen_v1.init.sql

```diff
@@ -0,0 +1,39 @@
+{{ header }}
+{% from "macros.sql" import core_clients_first_seen %}
+
+CREATE TABLE IF NOT EXISTS
+  `{{ first_seen_table }}`
+PARTITION BY
+  first_seen_date
+CLUSTER BY
+  sample_id
+OPTIONS
+  (require_partition_filter = FALSE)
+AS
+WITH
+  baseline AS (
+    SELECT
+      client_info.client_id,
+      DATE(MIN(submission_timestamp)) as first_seen_date,
+      MIN(sample_id) as sample_id
```

Comment:
I prefer to GROUP BY `sample_id, client_id` so that we can reference `sample_id` directly without an arbitrary aggregation function.